Add test API + 2 common utils

Mihullo opened the pull request:
Added:
1. Test suite of API at https://reqres.in/
2. Two common utilities: colored_messages, extract_from_json
3. Package name 'Fake' in requirements.txt

BurhanH commented on Utils/colored_messages.py:
```diff
@@ -0,0 +1,5 @@
+
+
+def yellow_on_red_back(text):
+    return f'\033[33m\033[41m\033[6m\033[1m >>> {text} <<< \033[0m'
```
Interesting!

BurhanH commented on requirements.txt:
```diff
@@ -1 +1,2 @@
 requests==2.27.1
+Faker
```
Do we need specify a version of faker library?

Mihullo replied:
I think it would be better to fix the version for the stability of the framework. I will mark the current version 13.13.0.

BurhanH commented on tests/test_full_reqres_api_MiK.py:
```diff
@@ -0,0 +1,168 @@
+
+TEST_DATA = {
+    'name': Fake().name(),
+    'job':  Fake().job(),
```
Nice! Using faker to generate test data!

BurhanH commented on tests/test_full_reqres_api_MiK.py:
```diff
@@ -0,0 +1,168 @@
+    def test_1_check_list_users_pages(self):
+        """Getting the default list of users. Determining the number of pages and users."""
+        response = requests.get(f'{BASE_URL}{ALL_USERS}')
+        self.assertEqual(response.status_code, HTTP_STATUS_OK, bug_report('Bad request total user page.'))
```
Now I got it! If test fail we will get a colored error message in terminal. Nice!

BurhanH commented on tests/test_full_reqres_api_MiK.py:
```diff
@@ -0,0 +1,168 @@
+        self.assertEqual(response.status_code, HTTP_STATUS_OK, bug_report('Bad request total user page.'))
+        total_user_pages = (response.json())['total_pages']
+        self.assertGreater(total_user_pages, 0, bug_report('The user database is not filled.'))
+        total_users = (response.json())['total']
```
Good!

BurhanH reviewed (commented):
Please review my comments!

BurhanH commented:
For some reason your PR has failures.
Please fix them!
Thanks @Mihullo

Mihullo commented:
**Fixed!**
Added: conftest.py -- Without it, Pytest does not see the 'Utils' package;
Deleted: __init.py__ from tests\ -- With it, PyTest does not import the 'rest_api_helper' module from tests\ and crashes test_get_characters.py.
